spec/selenium_spec.rb

```diff
@@ -100,6 +100,22 @@ module TestSessions
         expect(@session).to have_selector(:css, '.change_event_triggered', :match => :one)
       end
     end
+
+    describe "#path" do
+      before :each do
+        @session.visit('/with_js')
```

Comment:
does this need JS in anyway? if not it shouldn't use the /with_js test page -- also since #page is part of the node api this should be moved to the specs used by all drivers rather than just the selenium driver